api/types/webauthn/webauthn.pb.go

```diff
@@ -39,56 +39,6 @@ var _ = math.Inf
 // proto package needs to be updated.
 const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
-// User represents a WebAuthn user.
-// Used mainly to correlated a WebAuthn user handle with a Teleport user.
-type User struct {
-	// Teleport user ID.
-	TeleportUser         string   `protobuf:"bytes,1,opt,name=teleport_user,json=teleportUser,proto3" json:"teleport_user,omitempty"`
```

Comment:
It's pretty confusing to figure out which json tag is used, but it's actually the snake_case one. Confirmed by marshaling a webauthn.User (before removing) and by looking at the sqlite key (before applying the change).

```
$ select * from kv where key like '/webauthn/users/%';
> /webauthn/users/N2I4Y2QzZDAtYTU2OS00NDMzLWE1MzktYjkxZTgwZWQ3ZTVm|1711633239496051000||{"teleport_user":"llama"}|c0224976-44d1-4c06-9f1a-eed41a5e9bbb
> /webauthn/users/ZjAxM2QxNDQtODQ1ZS00OGEzLTkyMGItMjE4NTU2MzhmMjVm|1711634315315319000||{"teleport_user":"alpaca"}|e9a1527b-e33f-4011-a14f-1e97ce6f8d6c
```